<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>record</class>
 <widget class="QDialog" name="record">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>353</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>英雄榜</string>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>263</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正剑体简体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string> 英  雄  榜</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>63</y>
     <width>361</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="rank">
        <property name="font">
         <font>
          <family>邯郸-刘书锋太极体</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>名次</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="name">
        <property name="font">
         <font>
          <family>邯郸-刘书锋太极体</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>姓名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="score">
        <property name="font">
         <font>
          <family>邯郸-刘书锋太极体</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>分数</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="rank_1">
        <property name="font">
         <font>
          <family>邯郸-刘书锋太极体</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="name_1">
        <property name="font">
         <font>
          <family>邯郸-刘书锋太极体</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="score_1">
        <property name="font">
         <font>
          <family>邯郸-刘书锋太极体</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="rank_2">
        <property name="font">
         <font>
          <family>邯郸-刘书锋太极体</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="name_2">
        <property name="font">
         <font>
          <family>邯郸-刘书锋太极体</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="score_2">
        <property name="font">
         <font>
          <family>邯郸-刘书锋太极体</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="rank_3">
        <property name="font">
         <font>
          <family>邯郸-刘书锋太极体</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="name_3">
        <property name="font">
         <font>
          <family>邯郸-刘书锋太极体</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="score_3">
        <property name="font">
         <font>
          <family>邯郸-刘书锋太极体</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>301</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>清空</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>rank</zorder>
  <zorder>name</zorder>
  <zorder>score</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget_2</zorder>
  <zorder>layoutWidget_3</zorder>
  <zorder>score_3</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>horizontalSpacer</zorder>
  <zorder>title</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
